.github/workflows/backport_issue_check.yml

```diff
@@ -0,0 +1,20 @@
+name: Backport Issue Check
```

Comment:
Please update the workflow as well: https://github.com/zephyrproject-rtos/zephyr/pull/45880#pullrequestreview-1040626430

Reworked workflow: https://github.com/zephyrproject-rtos/zephyr-testing/blob/v9.9-branch/.github/workflows/backport_issue_check.yml

You can copy and paste the content of the linked workflow file above, which has been verified in the testing repo, into this PR.

Reply:
Done!